scripts/test

```diff
@@ -36,4 +41,17 @@ PACKAGES="$(find . -name '*.go' -print0 | xargs -0 -I{} dirname {} |  cut -f2 -d
 
 trap "rm -f /tmp/test-disk" EXIT
 
+# Try to acquire exclusive lock on $LOCK_FILE using file descriptor 200
+# If the lock is held by another process, wait and retry
+exec 200>"$LOCK_FILE"
+until flock -n 200; do
```

Comment:
NIT: Consider simplifying the retry logic with `flock -w`. That said, I am also fine with the current.

Reply:
Good idea. Updated, thank you.